collect_app/src/androidTest/java/org/odk/collect/android/database/helpers/InstancesDatabaseHelperTest.java

```diff
@@ -43,14 +47,13 @@ public void restoreRealDb() {
         FileUtils.copyFile(new File(DATABASE_PATH + TEMPORARY_EXTENSION), new File(DATABASE_PATH));
     }
 
-    @Parameterized.Parameters(name = "{0}")
+    @Parameterized.Parameters(name = "{1}")
     public static Iterable<Object[]> data() {
         return Arrays.asList(new Object[][] {
-                {"Downgrading from version with extra column drops that column", "instances_v7000_added_fakeColumn.db"},
```

Comment:
Please add the 7000 tables and tests back in. They're there to guarantee that downgrade with added and removed columns are always tested no matter the current target database version.

Reply:
My bad, fixed.